Dockerfile

```diff
@@ -17,8 +17,8 @@ WORKDIR /go/src/github.com/kubernetes-sigs/aws-ebs-csi-driver
 ADD . .
 RUN CGO_ENABLED=0 GOOS=linux go build -o bin/aws-ebs-csi-driver ./cmd/aws-ebs-csi-driver
 
-FROM registry.fedoraproject.org/fedora-minimal
+FROM alpine:3.8
 COPY --from=builder /go/src/github.com/kubernetes-sigs/aws-ebs-csi-driver/bin/aws-ebs-csi-driver /bin/aws-ebs-csi-driver
-RUN microdnf install -y e2fsprogs && microdnf clean all
+RUN apk add --no-cache ca-certificates e2fsprogs
```

Comment:
Just realize one existing issue, could you switch `RUN apk add...` with `COPY` so that docker build doesn't generate new mid layer each time we make changes to driver?

Reply:
I mean switching the line order to be like:
```
RUN apk add --no-cache ca-certificates e2fsprogs
COPY --from=builder /go/src/github.com/kubernetes-sigs/aws-ebs-csi-driver/bin/aws-ebs-csi-driver /bin/aws-ebs-csi-driver
```

Because each line in docker file generates a new layer of docker image, we could avoid keep generating the `apk add` layer if it is before `COPY` line

Reply:
Gotcha... Done